odfuzz/entities.py

```diff
@@ -109,10 +109,6 @@ def __init__(self, metadata, restrictions):
         self._metadata_string = metadata
         self._restrictions = restrictions
 
-        Config.init()
```

Comment:
what is the rationale for this removal? Also does not seems to be relevant to the bug itself.